app/helpers/application_helper/toolbar/cloud_subnets_center.rb

```diff
@@ -11,6 +11,7 @@ class ApplicationHelper::Toolbar::CloudSubnetsCenter < ApplicationHelper::Toolba
           'pficon pficon-add-circle-o fa-lg',
           t = N_('Add a new Cloud Subnet'),
           t,
+          :klass => ApplicationHelper::Button::CloudSubnetNew
```

Comment:
There's no need for a new button class here really. You should use `GenericFeatureButtonWithDisable` like so:

```ruby
:klass => ApplicationHelper::Button::GenericFeatureButtonWithDisable,
:options => {:feature => :create}
```